tests/integration/dynamic_ip_test.py

```diff
@@ -39,6 +39,7 @@
 from .testlib import bondlib
```

Comment:
Regarding the commit message, how about this to make the behavior change more explicit:

```
With systemd version 242, Fedora 31 will use persistent MAC address for
all virtual network interface (including linux bridges). Therefore the bridge will not use the MAC address of the first added port anymore.
```

If you agree with the new name for the test function, it needs to be updated in the commit message as well. For the comments, it needs to be `systems` (plural) after  `all` and `Fedora`.